etc/adminhtml/system.xml

```diff
@@ -84,6 +84,14 @@
                         <field id="enabled">1</field>
                     </depends>
                 </field>
+                <field id="single_sign_on" translate="label comment" type="select" sortOrder="75" showInStore="1" showInWebsite="1" showInDefault="1">
+                    <label>Enable Single Sign On</label>
+                    <comment><![CDATA[Allow your users to register/sign-in to both the TurnTo system and your website.]]></comment>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
```

Comment:
Also, add details to the readme and potentially link to the readme like I did in "Custom Configuration" below